src/stko/_internal/calculators/extractors/xtb_extractor.py

```diff
@@ -359,3 +364,15 @@ def _extract_electron_affinity(self, line: str) -> None:
         self.electron_affinity = float(
             string.group(0)  # type: ignore[union-attr]
         )
+
+    def _extract_total_sasa(self, line: str) -> None:
+        """Updates :attr:`total_sasa`.
+
+        Parameters:
+            line:
+                Line of output file to extract property from.
+
+        """
+        nums = re.compile(r"[+-]?\d+(?:\.\d+)?(?:[eE][+-]?\d+)?")
```

Comment:
a comment which describes what kinds of things this is meant to match could be nice

Reply:
Added.